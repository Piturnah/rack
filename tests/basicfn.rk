fn test_fn in end // can we ret successfully

fn main in "hello world\n" puts test_fn "success!\n" puts end
