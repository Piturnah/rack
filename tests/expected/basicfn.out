----STDOUT----
hello world
success!

----STDERR----
